dapp/src/components/buySell/ContractsTable.js

```diff
@@ -191,7 +221,11 @@ const ContractsTable = () => {
                     ? '-'
                     : `$${formatCurrency(estimation.effectivePrice, 2)}`}
                 </div>
-                <div className={`text-right w-18 ${redStatus ? 'red' : ''}`}>
+                <div
```

Comment:
@sparrowDom can we throw in a `pl-2` on these so that we don't end up with "-Amount too high"?

Reply:
Yup ofc

Reply:
Never before have I worked with a project / product manager that would be able to tell me what lines of code I need to modify 👊

Reply:
Probably best to ignore me most of the time 😬

Reply:
Oh no quite the oposite. Your helping is saving time and is welcome